packages/patternfly-3/react-console/src/ConsoleSelector/ConsoleSelector.js

```diff
@@ -0,0 +1,157 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { Grid, Row, Col, Form, FormGroup, Dropdown, MenuItem } from 'patternfly-react';
+import { Checkbox } from 'react-bootstrap';
```

Comment:
can be moved to the line above... import { Checkbox } from 'patternfly-react';

Reply:
Done, thanks. It was not available when initially writing this PR.